libs/glfw/build.zig

```diff
@@ -68,7 +70,21 @@ pub const pkg = std.build.Pkg{
     .source = .{ .path = thisDir() ++ "/src/main.zig" },
 };
 
+// TODO(self-hosted): HACK: workaround https://github.com/ziglang/zig/issues/12483
+//
+// Extracted from a build using stage1 from zig-cache/ (`cimport/c_darwin[_native].zig`)
+// Then find+replace `= ?fn` -> `= ?*const fn`
+fn cimportWorkaround() void {
+    const dest_dir = cwd.openDir(thisDir() ++ "/src", .{}) catch unreachable;
```

Comment:
This might be a stage2 Windows issue, but I had to make each of these `cwd` be verbosely set `std.fs.cwd()` for each usage here.

If I use this code as is, I get this error at compile time of `build.zig`:
```
C:\Program Files\Zig\Zig\lib\std\os\windows.zig:1756:20: error: unable to evalutate comptime expression
        .x86_64 => asm volatile (
                   ^~~
C:\Program Files\Zig\Zig\lib\std\os\windows.zig:1769:15: note: called from here
    return teb().ProcessEnvironmentBlock;
           ~~~^~
C:\Program Files\Zig\Zig\lib\std\fs.zig:2380:41: note: called from here
        return Dir{ .fd = os.windows.peb().ProcessParameters.CurrentDirectory.Handle };
                          ~~~~~~~~~~~~~~^~
C:\Users\Vulfox\repos\furcifer\.zigmod\deps\git\github.com\hexops\mach\libs\glfw\build.zig:4:23: note: called from here
const cwd = std.fs.cwd();
```

Reply:
yes this a window issue and already known. will pushing the fix soon